schema/workflow/subworkflow.json

```diff
@@ -3,15 +3,12 @@
     "$schema": "http://json-schema.org/draft-04/schema#",
     "title": "subworkflow schema",
     "type": "object",
+    "allOf": [
+        {
+            "$ref": "base_flowchart.json"
```

Comment:
@spectre007 - don't we already have a base flowchart unit that can be reused here?

Reply:
Resolving this per our discussion.